source/docs/03-metadata.xml

```diff
@@ -1393,156 +1397,470 @@
+                        </p>
+                        <p>This entry could be encoded as follows:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample089.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>A slightly more structured form would be:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample090.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>These transcriptions – as in the musical text – can also be marked by means of <gi scheme="MEI">add</gi>, <gi scheme="MEI">del</gi>, (see module MEI.edittrans) etc. (see <ptr target="#editTrans"/>) and assigned to a specific scribe by <att>hand</att> (see <gi scheme="MEI">handList</gi>):</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample091.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>For structuring purposes, it may sometimes be useful to separate entries made by a composer in the manuscript from those made by others:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample092.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>Under certain circumstances, <gi scheme="MEI">stamps</gi> can also be encoded under <gi scheme="MEI">addDesc</gi>.
```

Comment:
```suggestion
                        <p>Under certain circumstances, <gi scheme="MEI">stamp</gi> elements can also be encoded under <gi scheme="MEI">addDesc</gi>.
```

Reply:
This threw a warning about an element definition that cannot be retrieved since there is no `stamps` element.